src/rules/require-baseline.js

```diff
@@ -625,6 +628,48 @@ export default {
 					});
 				}
 			},
+
+			Selector(node) {
+				for (const child of node.children) {
+					const selector = child.name;
+
+					if (!selectors.has(selector)) {
+						continue;
+					}
+
+					const ruleLevel = selectors.get(selector);
```

Comment:
This can be undefined if someone is using an unknown selector. In that case, we just want to exit without reporting anything.

Reply:
Is this addressed by the check above?

```js
if (!selectors.has(selector)) {
  continue;
}
```

Reply:
Ah yes, sorry about that.